src/services/schemas/utilities.ts

```diff
@@ -231,6 +232,16 @@ export function getSchemaValueDefinitionIndex(definitions: Schema[], value: Sche
     return definitions.findIndex(definition => definition.type === 'block')
   }
 
+  try {
+    const parsedValue = parseUnknownJson(value)
+
+    if (isRecord(parsedValue) || Array.isArray(parsedValue)) {
+      return findObjectDefinitionIndex(definitions, parsedValue)
+    }
+  } catch {
+    // do nothing, use the raw value
+  }
```

Comment:
We don't need a try catch here do we? `parseUnknownJson()` already silences the error.

Reply:
Ah good point - updated